Setup_Tablets_Spotter und Setup_Tablets_Schusszettel

yummi2 opened the pull request:
Button Setup_Tablets_Spotter umbenannt und Button Setup_Tablets_Schusszettel dargestellt
 
https://gitlab.gras-it.de/hsrt/swt2/-/issues/1874

yummi2 committed:
Button Setup_Tablets_Spotter umbenannt und Button Setup_Tablets_Schusszettel dargestellt (Test)

mklemmingen reviewed (approved):
lgtm

mklemmingen reviewed (changes requested):
button added in offline, but not in online button underneath!

mklemmingen committed:
added a mirrored setup tablet schusszettel safeguard.
Now, If offline, the button is also greyed out.

for explanation:

    ngIf="!isOffline()" means: if the app has an internet connection, show a live, enabled Spotter tablet setup button

    else onlineSetupTables (template fallback): if offline, show a disabled placeholder version
